src/k8s-engine/k3sHelper.ts

```diff
@@ -1050,6 +1050,29 @@ export default class K3sHelper extends events.EventEmitter {
     return !await this.failureDueToNetworkProblem(target);
   }
 
+  /**
+   * Check if the given Kubernetes version requires the port forwarding fix
+   * (where we listen on a local port).
+   *
+   * @param version Kubernetes version; null if no Kubernetes will run.
+   */
+  static requiresPortForwarding(version: semver.SemVer | null): boolean {
```

Comment:
A bit of a nitpick here: maybe this would be clearer to read if it was `requiresPortForwardingFix`?

Reply:
Renamed as suggested in 480ff4322e6a1fbfb663eca7fd5f7152c8b90c35